src/styles/utilities/_text.scss

```diff
@@ -28,6 +28,50 @@ $lines: (1, 2, 3, 4, 5, 6, 7);
 	text-decoration: none;
 }
 
+.u-text-default-font {
+	font-family: $font-family-primary;
+}
+
+.u-text-4xl {
```

Comment:
Duplicate code. There's already a util available for this which is much cleaner since it generates the values ;) 

https://github.com/viaacode/hetarchief-client/blob/bec2220b7578ae9df82903b830d77973d360d7b6/src/styles/utilities/_text.scss#L93